cli/src/pcluster/templates/cw_dashboard_builder.py

```diff
@@ -10,6 +10,8 @@
 # limitations under the License.
 from collections import defaultdict, namedtuple
 
+import aws_cdk.aws_cloudwatch
```

Comment:
Remove this line
There is already:  `from aws_cdk import aws_cloudwatch as cloudwatch`